### ISSUE TITLE
Symbols errors

### ISSUE THREAD

**Issue by Fedorus** [EvanCugat/PokemonRPG#1]
Looks like symbol é  is replased by symbol й in rusian localization. Looks like VS creating files in different encodings. On first download from github file encoding isn`t unicode so this symbols vere lost. On pull files vere donwloaded as unicode and this created mismatches in comparion in file PokeMart.cs line 41. And it droped null exception in Player.cs line 68.

**Comment by EvanCugat** [comment]
Thanks a lot for your comment and for letting me know! I'm gonna research how to fix the Unicode symbols issue. I think I have successfully fixed the null exception issue, please let me know if you still encounter any null exceptions!

**Comment by EvanCugat** [comment]
I think I have fixed the Unicode issue. My solution is now configured as Unicode. If you could check and let me know if it's solved I'd really appreciate it!

**Comment by Fedorus** [comment]
Same issue, pulled project from github and received files with wrong encoding.
[http://hostthenpost.org/uploads/14ceacc5fdf99aee397f5cf60da653ea.jpg](url)

**Comment by EvanCugat** [comment]
This is so weird, I saved the project as Unicode, previously it was UTF-8. None of these work for Russian? :S

**Comment by EvanCugat** [comment]
Hey thanks a lot! I do not see what you changed however! It all looks the same?! lol

**Comment by Fedorus** [comment]
Created pull request. Fixed error. I changed symbol and saved files in UTF-8. It`s just a strange thing.. 
My ideas on this:
When you printed this letter it was given code #111 (number doesn`t matter) which have letter representation in your localisation. When you changed file encoding to Unicode symbol code is stayed #111 and you see it as normal. But russian localisation doesn`t have this code so I have error-symbol й. 
When I`ve replaced symbol it had code not yours #111 but from its UTF-8 ? representation. So it displayed as normal. Weird shit. It could be proven by looking in hex by i`m too lazy )))))
